docs/en/query_language/functions/ip_address_functions.md

```diff
@@ -113,5 +113,38 @@ LIMIT 10
 The reverse function of IPv6NumToString. If the IPv6 address has an invalid format, it returns a string of null bytes.
 HEX can be uppercase or lowercase.
 
+## IPv4ToIPv6(x)
+
+Takes a UInt32 number. Interprets it as an IPv4 address in big endian. Returns a FixedString(16) value containing the IPv6 address in binary format. Examples:
+
+``` sql
+SELECT IPv6NumToString(IPv4ToIPv6(IPv4StringToNum('192.168.0.1'))) AS addr
+```
+
+```
+┌─addr───────────────┐
+│ ::ffff:192.168.0.1 │
+└────────────────────┘
+```
+
+## cutIPV6(x, cutIPv6, cutIPV4)
```

Comment:
* "V" and "v" look weird together
* I'd rename the arguments to something like bitsToCutForIPv4